tests/test-user.js

```diff
@@ -571,6 +573,52 @@ describe('User', () => {
       });
     });
 
+    it('should log error and return undefined when GET OVH returns 500 error', (done) => {
+      utils.cleanMocks();
+      utils.mockUsers();
+      utils.mockSlack();
+      utils.mockOvhTime();
+
+      const getOvhEmailCreation = nock(/.*ovh.com/)
+        .get(/^.*email\/domain\/.*\/account\/.*/)
+        .reply(500);
+
+      knex('users').insert({
+        username: 'utilisateur.nouveau',
+        secondary_email: 'utilisateur.nouveau.perso@example.com',
+      }).then(async () => {
+        const shouldBeUndefined = await createEmailAddresses();
+        getOvhEmailCreation.isDone().should.be.true;
+        should.not.exist(shouldBeUndefined);
```

Comment:
Si c'est plus simple vous pouvez aussi l'écrire à l'inverse :  `cronOutcome.should.not.exist` ou un truc comme ça

Reply:
la variable ` cronOutcome` étant undefined (produit par l'erreur levée), on ne peut pas accèder aux méthodes `.should`, l'astuce est d'utiliser une variable `should` importée plus haut dans le fichier

on a été inspirés par [ce stackoverflow](https://stackoverflow.com/a/19214161/3535853)

Reply:
Ah bah oui :smile: